### PR TITLE
migration status tracker

### PR DESCRIPTION
**Pull request by iwolkov**
Migration status tracker added.

### CODE REVIEW

File: ide-migration/migration-statistic.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  <!-- Twitter Bootstrap with Theme Support -->
+  <link href="/services/v4/js/theme/resources.js/bootstrap.min.css" rel="stylesheet">
+  <script src="/webjars/bootstrap/3.3.7/js/bootstrap.min.js" type="text/javascript"></script>
+
+  <!-- MessageHub -->
+  <script src="/services/v4/web/ide-core/ui/message-hub.js" type="text/javascript"></script>
+
+  <!-- Custom IDE Styles -->
+  <link href="/services/v4/js/theme/resources.js/ide.css" rel="stylesheet" type="text/css" />
+  <link href="data:;base64,iVBORw0KGgo=" rel="icon">
```
Comment (StanZGenchev) [inline ide-migration/migration-statistic.html:38]: Replace with `<link rel="icon" sizes="any" href="data:;base64,iVBORw0KGgo=">`

**Review by StanZGenchev — APPROVED** [review]

File: ide-migration/server/migration/api/track-service.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  addEntry(status) {
+    this.setupTable()
+    try {
+      entryInstance = migrationsTable.insert({
+        executedBy: userName,
+        startedOn: Date.now(),
+        lastUpdated: Date.now(),
+        status: status
+      });
+      const entry = migrationsTable.find(entryInstance)
```
Comment (ThuF) [inline ide-migration/server/migration/api/track-service.js:101]: isn't `entryInstance` equals to `entry.id`? If that's the case, then t he `migrationsTable.find()` is not needed.

**Review by ThuF — APPROVED** [review]
+1

**Comment by ThuF** [comment]
@iwolkov should we merge the PR? Are we waiting for more changes?

**Comment by StanZGenchev** [comment]
@ThuF No, we are waiting for approval from @vmutafov

File: ide-migration/server/migration/api/track-service.js (hunk 2 above)
Comment (iwolkov) [reply ide-migration/server/migration/api/track-service.js:101]: Yes, you are right

**Review by ThuF — APPROVED** [review]

**Pull request merged**